### PR TITLE
Microverse Day 3 portfolio

### PR DESCRIPTION
**Pull request by Jonathan-kiprono**
Edit HTML file
Edit CSS file 
Add Images Folder

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,3 +1,429 @@
-h1 {
-  color: green;
+body {
+  background: #e5e5e5;
+  background-image: ./hello/images/header-shapes.png;
```
Comment (Tufahel) [inline style.css]: - [ ] This line of code seems not working as it is in the wrong format. Kindly remove this or fix this issue.

File: style.css [hunk 2]
```diff
@@ -1,3 +1,429 @@
...
+  display: flex;
+  width: 170.17px;
+  height: 20px;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: baseline;
+  padding-bottom: 136px;
+  list-style-type: none;
+  padding-left: 15px;
+}
```
Comment (Tufahel) [inline style.css]: - [ ] Your UI looks great and almost perfect. 👍 Just you need to fix the alignment of the icons with the `title` and `paragraph` sections. The icons seem a bit outside of the alignment. Kindly fix this to match the given figma design.

- Your design
- image

- Given Figma Design
- image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Jonathan-kiprono  :wave: 

Well done. :+1:  
You need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing.
✔️ Commit message is professional.
✔️ Readme is professional.


## Required Changes ♻️

> Kindly check the inline comments for review.

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -1,3 +1,428 @@
-h1 {
-  color: green;
+body {
```
Comment (huseyinbiyik) [inline style.css:1]: [OPTIONAL] Good job with the requested changes. 👍  But let's fix the overflow issue, kindly remove the 8px margin around the body and the page will look perfect 👍 
Screenshot 2023-03-25 at 2 05 44 AM

**Review by huseyinbiyik — APPROVED** [review]
Hi @Jonathan-kiprono 

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- PR description is descriptive 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
- Kindly remove this instructive text below from your README and a software license. The link on your license section is broken. You can follow the instructions at the bottom of [README](https://github.com/microverseinc/readme-template/blob/master/README.md) template to add a license.
Screenshot 2023-03-25 at 2 10 58 AM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**